src/functional/ffront/fbuiltins.py

```diff
@@ -69,8 +70,20 @@ def __gt_type__(self):
     )
 )
 
+where = BuiltInFunction(
```

Comment:
The reason we have picked "where" is because that is how the equivalent function in numpy is called (https://numpy.org/doc/stable/reference/generated/numpy.where.html).